src/tests/commonTests.ts

```diff
@@ -0,0 +1,79 @@
+import { newE2EPage } from "@stencil/core/testing";
+import * as pd from "@stencil/core/dist/testing/puppeteer/puppeteer-declarations";
```

Comment:
If it is possible, would probably be better to just import the type you need like:

```
import { E2EPage } from "@stencil/core/dist/testing/puppeteer/puppeteer-declarations";
```

Reply:
@jcfranco want to update our file in our repo? :)